sphinx/source/members/constitution.rst

```diff
@@ -0,0 +1,64 @@
+Constitution
+============
+
+The :term:`constitution` defines the set of rules and conditions (as a Lua script) that members must follow for their proposals to be accepted. For example, in a CCF network governed by `4` members, a strict majority constitution would only execute proposals once `3` members (`4/2 + 1`) have voted for that proposal.
+
+Votes for the proposal are evaluated by the constitution's ``pass`` function. If the `pass` function returns ``true``, the vote is passed, and its consequences are applied to the KV in a transaction.
+
+Examples of constitution include (see :ref:`Models` for further details):
+
+- Strict majority (`simple constitution`_) that implements a "one-member, one-vote" constitution, with a majority rule. Votes on so-called sensitive tables, such as the one containing the constitution itself, require unanimity.
+- Operating member + strict majority (`operating member constitution`_) that extends the "strict majority" constitution by defining an operating member allowed to add new nodes to the network, retire existing ones and allow new versions of the code.
+
+Once the initial set of members have agreed on a constitution, the corresponding Lua file can be given to operators to create a new network (see :ref:`Starting a New Network`).
```

Comment:
I think we should do a pass on all of the "(see XXX)" entries and either remove them, or make them inline links. For instance this one could be "to `create a new network <Starting a New Network>`_ or whatever the `.rst` syntax is. This is less intrusive for the reader, visually and mentally, than the parenthesthesestheseth.

Reply:
I agree with replacing the "(see XXX)" with inline links.